fontforgeexe/collabclientui.c

```diff
@@ -471,30 +471,31 @@ gpointer getSessionCallbacksObject()
 void collabclient_notifySessionJoining( cloneclient_t *cc, FontViewBase* fv )
 {
 #ifdef BUILD_COLLAB
-    
-    printf("AAA collabclient_notifySessionJoining() fv:%p\n", fv);
-    g_signal_emit_by_name( getSessionCallbacksObject(), "joining", fv );
-    
+    if (cc) {
```

Comment:
Is there ever a case in which it would be okay for cc to be NULL in this context? If not, we ought to have a menacing error message in the else block.